### PR TITLE
Fix/wz 1661 sluiting

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/app/components/shared/ConfirmScaffoldForm/utils/createValuesObject.tsx [hunk 1]
```diff
@@ -5,23 +5,28 @@ import { Field } from "../../Form/ScaffoldField"
...
-      <ArrayFieldList fields={ v as Array<Record<string, string>> } /> :
-    type === "ComplexSelectField" || type === "ComplexRadioFields" ?
-      (v as Record<string, string>)[(props as { optionLabelField: string }).optionLabelField] :
-    props.hasOwnProperty("options") ?
-      (props as { options: Record<string, unknown> }).options[v as string] :
-      v
+  const value = (type: string, props: any) => {
+    if(type === "ArrayField") {      
+      return <ArrayFieldList fields={ v as Array<Record<string, string>> } />
+    }else if(type === "ComplexSelectField" || type === "ComplexRadioFields"){
```
Comment (remyvdwereld) [inline src/app/components/shared/ConfirmScaffoldForm/utils/createValuesObject.tsx:17]: Space before else?

File: src/app/components/shared/ConfirmScaffoldForm/utils/createValuesObject.tsx [hunk 2]
```diff
@@ -5,23 +5,28 @@ import { Field } from "../../Form/ScaffoldField"
...
-  const value =
-    type === "ArrayField" ?
-      <ArrayFieldList fields={ v as Array<Record<string, string>> } /> :
-    type === "ComplexSelectField" || type === "ComplexRadioFields" ?
-      (v as Record<string, string>)[(props as { optionLabelField: string }).optionLabelField] :
-    props.hasOwnProperty("options") ?
-      (props as { options: Record<string, unknown> }).options[v as string] :
-      v
+  const value = (type: string, props: any) => {
+    if(type === "ArrayField") {
```
Comment (remyvdwereld) [inline src/app/components/shared/ConfirmScaffoldForm/utils/createValuesObject.tsx:15]: Space after if?

File: src/app/components/shared/ConfirmScaffoldForm/utils/createValuesObject.tsx [hunk 3]
```diff
@@ -5,23 +5,28 @@ import { Field } from "../../Form/ScaffoldField"
...
-      (props as { options: Record<string, unknown> }).options[v as string] :
-      v
+  const value = (type: string, props: any) => {
+    if(type === "ArrayField") {      
+      return <ArrayFieldList fields={ v as Array<Record<string, string>> } />
+    }else if(type === "ComplexSelectField" || type === "ComplexRadioFields"){
+      return(v as Record<string, string>)[(props as { optionLabelField: string }).optionLabelField]
+    } else if (props.hasOwnProperty("options")) {
+      return (props as { options: Record<string, unknown> }).options[v as string] 
+    } else if( name.includes("type_result") ) {
```
Comment (remyvdwereld) [inline src/app/components/shared/ConfirmScaffoldForm/utils/createValuesObject.tsx:21]: `} else if (name.includes("type_result")) {`

**Review by remyvdwereld — APPROVED** [review]

File: src/app/components/shared/ConfirmScaffoldForm/utils/createValuesObject.tsx (hunk 3 above)
Comment (remcohoff) [reply src/app/components/shared/ConfirmScaffoldForm/utils/createValuesObject.tsx]: Sorry for the mess, fixed it

**Pull request merged**